### DISCUSSION TITLE
Jinja Template EMRContainerOperator in configuration_overrides  parameter unavailable

### DISCUSSION THREAD

**Discussion by joserivera1990** [apache/airflow#28748] [Q&A] [answered]
Hi Everyone,

I'm using Airflow with the version 2.2.2 in Amazon MWAA.

I would like to use the jinja template for the operator **EMRContainerOperator** in the **configuration_overrides**  parameter but, I checked the documentation and for that parameter it is unavailable.

https://airflow.apache.org/docs/apache-airflow-providers-amazon/2.2.0/_api/airflow/providers/amazon/aws/operators/emr_containers/index.html

![image](https://user-images.githubusercontent.com/16911323/210826536-40268d62-2338-45b9-9557-fcf4a8b0e81d.png)

¿Could you please add that parameter to the jinja template? ¿Is it possible?

On the other hand, I understand that I can use variables freely inside the execute() methods of the operators but, I don't know how to do it. ¿Could you please give me an idea about it? Maybe some code segment.

https://airflow.apache.org/docs/apache-airflow/stable/best-practices.html#airflow-variables

Thanks.

Regards.

**Comment by Taragolis** [comment, marked as answer]
> ¿Could you please add that parameter to the jinja template? ¿Is it possible?

Yes it is possible. Airflow is Open Source Software so everyone could make a PR but it could take some time: review and wait until next version of amazon-provider released 

As workaround you could overwrite this class attribute

```python

import airflow.providers.amazon.aws.operators.emr from EmrContainerOperator

EmrContainerOperator.template_fields = {"configuration_overrides", *EmrContainerOperator.template_fields}
```

or create new class based on `EmrContainerOperator`

```python

import airflow.providers.amazon.aws.operators.emr from EmrContainerOperator

class AwesomeEmrContainerOperator(EmrContainerOperator)
    template_fields = {"configuration_overrides", *EmrContainerOperator.template_fields}

```

**Reply by joserivera1990** [reply]
Hi @Taragolis, 
The two workaround options work fine  for me :)
Thank you for your kind help.
